### PR TITLE
Add a "gold runs" button to nearline monitoring pages

### PR DESCRIPTION
**Pull request by tannerbk**
Also a couple unrelated bug fixes.

### CODE REVIEW

File: minard/templates/nearline_summary.html [hunk 1]
```diff
@@ -164,9 +164,7 @@
 			        <th class="danger"> Not Launched </th>
 		                <th> {{ program }} </th>
                                 <th><a href="{{ url_for("nearline", run=run) }}">{{ run }}</a></th>
-                                <th>
-                                    <a href="{{ "/monitoring/nearline/logs/%i/%s.log" % (run,program) }}">{{ program }}.log</a>
-                                </th>
+                                <th> - </th>
```
Comment (tlatorre-uchicago) [inline minard/templates/nearline_summary.html]: Is there a reason to delete the link to the log files from the nearline summary page?

Comment (tannerbk) [reply minard/templates/nearline_summary.html]: This block is for the "not launched" jobs, so the logfile doesn't exist.

**Comment by tannerbk** [comment]
Looks like there is an issue with the overview page, I'll put in a fix.

**Comment by tlatorre-uchicago** [comment]
One thing I noticed is that if you go the page, hit "Show only gold runs" and then change the limit to 500 it goes back to gold=0.

**Comment by tannerbk** [comment]
Yah that was intentional, but I can make it so you have to hit "show all runs" to go back. Probably that would be more intuitive.

**Comment by tlatorre-uchicago** [comment]
Yeah, I think that's better since the first thing I did to test it was to hit "Show Only Gold Runs" and since none show up I tried to increase the limit.

I've noticed one other behavior that I'm not sure if it's intentional or not. Without selecting gold runs I can change the limit from 25 -> 500 and everything works ok. When I select just golden runs and then change the limit it shows nothing. If I manually set a run range it seems to work.

**Comment by tannerbk** [comment]
It looks like the last run on the golden run list is 106311 which is more than 500 runs ago -- I think that explains what you saw?

**Comment by tannerbk** [comment]
By the way the reason I added this is so we can look at the results of the channel flags process for all the golden runs in the past. I've been working on getting all the old zdabs (prior to nearline code being implemented) processed and the ratdb tables pushed.

**Comment by tlatorre-uchicago** [comment]
Ah ok, didn't realize there was a big gap in the golden runs there.

**Pull request merged**